docs/coding_rules.md

```diff
@@ -11,7 +11,9 @@ Naming conventions:
  * Always write ID with uppercase, this overwrites the other rules.
     * Do not write Id or id.
 
-Allman style is used for all brackets.
+Allman style is used for all brackets in .java files.
```

Comment:
Should include header for Bracket Styles since we now have multiple points about that.